src/mainContainer.css

```diff
@@ -0,0 +1,77 @@
+
+.container{
+    width: 90%;
+    margin: 5px auto;
+}
+.container .main-courses {
+    padding-left: 30px;
+    padding-right: 30px;
+    padding-top: 50px;
+    width: inherit;
+    
+}
+  
+.main-courses p {
+    font-size: 18px;
+    font-family: sans-serif;
+    margin-left: 5px;
+    
+  }
+.main-courses .course-list {
+    width: 100%;
+    height: auto;
+    margin-bottom: 20px;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+ 
+  .main-courses .course-list div {
```

Comment:
please update the tabs in the whole file to be consistent.

Reply:
i didn't understand this , in our second meeting for code review you told me to make the wrap because the origin website show them as slider bar

Reply:
I mean the spaces at the beginning of the line.